packages/protobuf-test/src/message.test.ts

```diff
@@ -214,7 +214,7 @@ describe("Message.fromJsonString()", function () {
     expect(() =>
       TestAllTypesProto3.fromJsonString("this is not json")
     ).toThrowError(
-      "cannot decode protobuf_test_messages.proto3.TestAllTypesProto3 from JSON: Unexpected token h in JSON at position 1"
+      "cannot decode protobuf_test_messages.proto3.TestAllTypesProto3 from JSON"
```

Comment:
The test is named "raises wrapped error on parse error". We need to keep testing that the root error message is included:

```suggestion
      /^cannot decode protobuf_test_messages\.proto3\.TestAllTypesProto3 from JSON: Unexpected token '?h'? in JSON at position 1$/
```